runcellpose.py

```diff
@@ -41,17 +42,18 @@
 
 """
 
-MODE_CELLS = "Cells"
-MODE_NUCLEI = "Nuclei"
-MODE_CUSTOM = "Custom"
+
+
+model_dic = models.MODEL_NAMES
+model_dic.append('custom')
 
 
 class RunCellpose(ImageSegmentation):
     category = "Object Processing"
 
     module_name = "RunCellpose"
 
-    variable_revision_number = 2
+    variable_revision_number = 3

```

Comment:
If we've bumped versions, it's a good time to add an upgrade_settings to this module :)